infoenergia_api/contrib/reports.py

```diff
@@ -0,0 +1,128 @@
+import aioredis
+import aiohttp
+import asyncio
+
+import ssl
+import json as jsonlib
+from sanic.response import json
+from sanic.log import logger
+
+class beedataApi(object):
+
+    def __init__(self):
+        from infoenergia_api.app import app
```

Comment:
like django, we implemented the configuration of the app like a singleton object, so we can access to the settings without load any module or app
to get the configuration anywhere in the code: 
```python
from config import config
``` 
and `config` will have all settings :+1: